### DISCUSSION TITLE
Workflow rendering doesn't work with yaml files

### DISCUSSION THREAD

**Discussion by travisgroth** [community/community#25015] [Actions]
<p>Hi,</p>
<p>With the transition to YAML based workflows, it seems like the visualizations that have been a prominent part of GitHub Actions marketing are now gone:  </p>
<p>Example legacy syntax: <a href="https://github.com/actions/docker/blob/master/.github/main.workflow" rel="nofollow noopener">https://github.com/actions/docker/blob/master/.github/main.workflow</a></p>
<p>New syntax: <a href="https://github.com/travisgroth/actions-test/blob/master/.github/workflows/main.yaml" rel="nofollow noopener">https://github.com/travisgroth/actions-test/blob/master/.github/workflows/main.yaml</a></p>
<p>When is this being added for yaml based workflows?  This is especially important when using ‘needs’ on longer workflows.</p>
<p>Thanks</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/travisgroth">@travisgroth</a>,</p>
<p>Thank you for being here!  Yes, the visual editor was only available for the HCL-based Actions for now.</p>
<p>We have heard a lot of feedback about bringing it back for the new YML-based Actions.</p>
<p>We can’t make any promises if/when that will happen though, but I’ll be sure to add your name to that list!</p>

**Comment by travisgroth** [comment]
<p>Hi Andrea,</p>
<p>Thank you for the answer.  Does GitHub realize that the visual editor/viewer is  <strong>very</strong> prominent in basically all material promoting Actions up until now?  It is also a feature in most competing products.  Silently removing it was inevitably going to get you a lot of feedback.  To be honest, it is shocking and disappointing that you’re not already working on it or at least commited to it. </p>
<p>This is the kind behavior that will discourage early adopters and instill distrust in other products such as Respositories.  </p>

**Comment by thejoebourneidentity** [comment, marked as answer]
<p>Thanks for the feedback <a class="mention" href="/travisgroth">@travisgroth</a>, I’m a product manager on GitHub Actions. This is something we’re keeping an eye on, but we have no timeline for bringing it back right now. Please let us know if you have any other feedback!</p>

**Comment by Datamance** [comment]
<p>It might make sense for you guys to get in contact with the folks over on the LinkedIn Learning team and coordinate on an update for the “Learning GitHub Actions” course - almost all of that material is basically useless now.</p>
<p>Really hoping you guys bring the visual editor back - it’s sorely missed!</p>

**Comment by prescottprue** [comment]
<p>Would also love to see this come back. A big reason why my team was interested in Github actions over other platforms was this ability to visualize flows.</p>
<p>Can I also have my name added to the list of folks getting updates about when this will be back?</p>

**Comment by softwareguy74** [comment]
<p>So you guys did all that work to create that visual editor AND use it in all the marketing material, only to completely drop it on launch?</p>

**Comment by BMayhew** [comment]
<p>Found this post after googling, after trying to follow Readme from current actions in the GitHub marketplace that all referred to the visual editor. I almost gave up…</p>

**Comment by bsridatta** [comment]
<p>The only place that talks about this. So weird. I thought I was not able to find it. So it has been removed :expressionless:</p>

**Comment by Misiu** [comment]
<p>The visual editor is still shown in all the pages on GitHub - <a href="https://github.com/features/actions" rel="nofollow noopener">https://github.com/features/actions</a></p>
<p>It’s a really missing feature.</p>
<p>Even if you don’t want to allow editing at least give users an option to show a visual representation based on YAML.</p>
<p>It is easier to show something visually that in code.</p>
